### PR TITLE
Implement MVVM for the GlobalAppearance page

### PR DESCRIPTION
**Pull request by PankajBhojwani**
## Summary of the Pull Request
Implements a `GlobalAppearanceViewModel` for the GlobalAppearance page in the SUI

## PR Checklist
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [x] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Validation Steps Performed
Global appearance page still works

### CODE REVIEW

**Review by carlos-zamora — APPROVED** [review]
Looks good to me! Thanks!

**Comment by carlos-zamora** [comment]
Oh, can you clean up the PR description? It has a lot of checkboxes that are unnecessary. Just delete those lines.

**Review by carlos-zamora — CHANGES_REQUESTED** [review]
Blocking until 1.15 forks (I approve the PR in its current state though )

**Review by carlos-zamora — APPROVED** [review]

File: src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml [hunk 1]
```diff
@@ -43,72 +43,72 @@
...
+                          ItemsSource="{x:Bind ViewModel.TabWidthModeList, Mode=OneWay}"
+                          SelectedItem="{x:Bind ViewModel.CurrentTabWidthMode, Mode=TwoWay}"
                           Style="{StaticResource ComboBoxSettingStyle}" />
             </local:SettingContainer>
 
             <!--  Disable Animations  -->
             <!--  NOTE: the UID is "DisablePaneAnimationsReversed" not "DisablePaneAnimations". See GH#9124 for more details.  -->
             <local:SettingContainer x:Uid="Globals_DisableAnimationsReversed">
-                <ToggleSwitch IsOn="{x:Bind local:Converters.InvertBoolean(State.Globals.DisableAnimations), BindBack=State.Globals.SetInvertedDisableAnimationsValue, Mode=TwoWay}"
+                <ToggleSwitch IsOn="{x:Bind local:Converters.InvertBoolean(ViewModel.GlobalSettings.DisableAnimations), BindBack=ViewModel.GlobalSettings.SetInvertedDisableAnimationsValue, Mode=TwoWay}"
```
[the remark this answers is not in the document]
Comment (PankajBhojwani) [reply src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml]: Fixed!

**Pull request merged**